muted-tests.yml

```diff
@@ -148,6 +148,9 @@ tests:
 - class: org.elasticsearch.xpack.inference.InferenceRestIT
   method: test {p0=inference/30_semantic_text_inference/Calculates embeddings using the default ELSER 2 endpoint}
   issue: https://github.com/elastic/elasticsearch/issues/117349
+- class: org.elasticsearch.xpack.inference.InferenceRestIT
+  method: test {p0=inference/30_semantic_text_inference_bwc/Calculates embeddings using the default ELSER 2 endpoint}
+  issue: https://github.com/elastic/elasticsearch/issues/117349
```

Comment:
We muted the default ELSER YAML tests for now because they download the real ELSER model, which often times out (or just fails) in CI. This was leading to Elasticmachine doing bad things, like auto-muting all of the inference YAML tests 😬

Reply:
i don't understand, it is muted in main too? If so let's remove the test entirely

Reply:
Yes, it's muted in main as well. We originally muted the test because @davidkyle & I wanted to stabilize it by adding [model download retries to the rest client when running YAML tests](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/inference/src/yamlRestTest/java/org/elasticsearch/xpack/inference/InferenceRestIT.java#L35). However, we haven't had the time to address that yet.

Reply:
ok thanks